src/app/inventory/store/stats.ts

```diff
@@ -158,7 +162,8 @@ export function buildStats(
 }
 
 /**
- * This builds a class items stat values from its masterwork and mod sockets.
+ * This builds a class item's stat values from its masterwork and mod sockets.
+ * to-do: can we get rid of this once we aren't using Live Stats for armor?
```

Comment:
Yes,  the whole reason this is here is because we know what the base stat values are (spoiler it's 0)